products/sle/main.pm

```diff
@@ -380,7 +380,10 @@ sub load_x11regression_documentation {
         loadtest "x11regressions/libreoffice/libreoffice_mainmenu_components";
         loadtest "x11regressions/libreoffice/libreoffice_recent_documents";
         loadtest "x11regressions/libreoffice/libreoffice_default_theme";
-        loadtest "x11regressions/libreoffice/libreoffice_pyuno_bridge";
+        loadtest "x11regressions/evolution/evolution_prepare_servers";
+        if (get_var("VERSION") =~ /12/) {
```

Comment:
Yes, IMHO it's better to unconditionally trigger this. When it will fail on SLE 15 then we will need to think about how to handle it there. Maybe we will just use a version specific repository when we get to that point.